### PR TITLE
Fix bunch of typos

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: deploy/helm/charts/values.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ lvmNode:
     type: RollingUpdate
   annotations: {}
   podAnnotations: {}
-  # This can be configured to run on various different k8s ditributions like
+  # This can be configured to run on various different k8s distributions like
```
Comment (prateekpandey14) [inline deploy/helm/charts/values.yaml:36]: This will require a chart version bump as well in Chart.yaml file.

**Review by prateekpandey14 — COMMENTED** [review]
@surajssd thanks for the PR, 1 minor comment

File: deploy/helm/charts/values.yaml (hunk 1 above)
Comment (surajssd) [reply deploy/helm/charts/values.yaml:36]: Done!

**Review by prateekpandey14 — APPROVED** [review]

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Comment by pawanpraka1** [comment]
Thanks for this PR @surajssd .

**Pull request merged**